theano/sparse/tests/test_basic.py

```diff
@@ -2105,7 +2105,6 @@ def op_with_fixed_index(x):
 
     def test_GetItem2D(self):
         scipy_ver = [int(n) for n in scipy.__version__.split('.')[:2]]
-        assert scipy_ver >= [0, 11]
```

Comment:
This made our daily buildbot fail this test as it use scipy 0.9. I ran the test with that version of scipy and they passed. So I think it is ok.